FinTExWebSite/public/map.html

```diff
@@ -0,0 +1,474 @@
+            </div>
+
+            <!-- Collect the nav links, forms, and other content for toggling -->
+            <div class="collapse navbar-collapse" id="bs-example-navbar-collapse-1">
+                <ul class="nav navbar-nav navbar-right" sytle="padding-top:0px;">
+                    <!-- <li class="hidden">
+                        <a href="#page-top"></a>
+                    </li> -->
+                    <li class="page-scroll">
+                        <a href="/#membership">Membership</a>
+                    </li>
+                    <li class="page-scroll">
+                        <a href="/#fintech-chicago">FinTech in Chicago</a>
+                    </li>
+                    <li class="page-scroll">
+                        <a href="/#events">Events</a>
+                    </li>
+                    <li class="page-scroll">
+                        <a href="/#about">About</a>
+                    </li>
+                    <li class="page-scroll">
+                        <a href="/#contact">Contact</a>
+                    </li>
+                    <li class="page-scroll">
+                        <a href="/map">Map</a>
+                    </li>
+                    <li class="page-scroll">
+
+                        <form action="https://docs.google.com/forms/d/1Eayxd5wN2tU0MpbrMleeLM3Nsz0RJoMofxE3kDYYxPo/formResponse" method="POST" id="ss-form" target="_blank" onsubmit="">
+                            <div class="ss-form-question errorbox-good" role="listitem">
+                                <div dir="auto" class="ss-item ss-item-required ss-text">
+                                    <div class="ss-form-entry">
+                                        <input type="text" name="entry.1928406774" placeholder="Your email address"  value="" class="ss-q-short" id="entry_1928406774" dir="auto" aria-label="Email  " aria-required="true" required="" title="">
+                                        <input type="hidden" name="draftResponse" value="[,,&quot;-8793509467677659677&quot;]
+">
+                                        <input type="hidden" name="pageHistory" value="0">
+
+                                        <input type="hidden" name="fvv" value="0">
+
+                                        <input type="hidden" name="fbzx" value="-8793509467677659677">
+
+                                        <input type="submit" name="submit" value="Subscribe" id="ss-submit" class="btn btn-success btn-md">
+
+                                        <div class="error-message" id="1637753161_errorMessage"></div>
+
+                                    </div>
+                                </div>
+                            </div>
+                        </form>
+
+                    </li>
+
+
+
+                </ul>
+            </div>
+            <!-- /.navbar-collapse -->
+        </div>
+        <!-- /.container-fluid -->
+    </nav>
+
+    <a id="downloadAnchorElem" style="display:none"></a>
+<style>
```

Comment:
We can move these styles into the `main.css` file once we're happy with how everything is loading on the staging server.